### ISSUE TITLE
Aurora Data-Api Postgres fields throwing cast error

### ISSUE THREAD

**Issue by seawatts** [typeorm/typeorm#7328]
## Issue Description
Several field types do not work properly, including `uuid`, `jsonb`, `money`, `enums` etc. are not being cast correctly with the Aurora Data API. 

### Expected Behavior
Be able to make calls with the `aurora-data-api-driver` with the above types.

### Actual Behavior

```
"code": "BadRequestException",
    "message": "ERROR: column \"foreign_id\" is of type uuid but expression is of type character varying\n  Hint: You will need to rewrite or cast the expression.
```

### Steps to Reproduce
Create a simple program that uses `aurora-data-api-pg` driver with https://github.com/ArsenyYankovsky/typeorm-aurora-data-api-driver

### My Environment

| Dependency          | Version  |
| ---                 | ---      |
| Operating System    |     Mac OSX     |
| Node.js version     | v15.x | 
| Typescript version  | v4.x | 
| TypeORM version     | v0.2.29 | 

### Additional Context

The related issues can be seen in the https://github.com/ArsenyYankovsky/typeorm-aurora-data-api-driver repo

https://github.com/ArsenyYankovsky/typeorm-aurora-data-api-driver/issues/44
https://github.com/ArsenyYankovsky/typeorm-aurora-data-api-driver/issues/48
https://github.com/ArsenyYankovsky/typeorm-aurora-data-api-driver/issues/57

There has been a fix created downstream in the [Data API Client](https://github.com/jeremydaly/data-api-client) to support casting. [Details here](https://github.com/jeremydaly/data-api-client#type-casting). However, TypeORM isn't internally passing type info to https://github.com/ArsenyYankovsky/typeorm-aurora-data-api-driver in order to send the cast information to https://github.com/jeremydaly/data-api-client

### Relevant Database Driver(s)

- [x] `aurora-data-api`
- [x] `aurora-data-api-pg`
- [ ] `better-sqlite3`
- [ ] `cockroachdb`
- [ ] `cordova`
- [ ] `expo`
- [ ] `mongodb`
- [ ] `mysql`
- [ ] `nativescript`
- [ ] `oracle`
- [ ] `postgres`
- [ ] `react-native`
- [ ] `sap`
- [ ] `sqlite`
- [ ] `sqlite-abstract`
- [ ] `sqljs`
- [ ] `sqlserver`

### Are you willing to resolve this issue by submitting a Pull Request?

- [ ] Yes, I have the time, and I know how to start.
- [ ] Yes, I have the time, but I don't know how to start. I would need guidance.
- [ ] No, I don't have the time, although I believe I could do it if I had the time...
- [x] No, I don't have the time and I wouldn't even know how to start.

**Comment by carlocorradini** [comment]
Any update? 😥

**Comment by rrgarciach** [comment]
Having same issue here.

**Comment by carlocorradini** [comment]
> Having same issue here.
> This issue seems to be addressed on PR #7483
> Everything seems to be ready to be merged! Let's go for it!

@rrgarciach 

See [this](https://github.com/ArsenyYankovsky/typeorm-aurora-data-api-driver/issues/57) issue for more information and a temporary fix.

*While we're waiting for the TypeORM pull request to be merged and a new release, you can use the fork of TypeORM, branch data-api-prepare-hydrate-packaged.*

1. Uninstall TypeORM
2. Install ``github:ArsenyYankovsky/typeorm#data-api-prepare-hydrate-packaged``

**Comment by AlexMesser** [comment]
implemented in #7483
